OSDOCS-5272: adds zstream update 4.12.3 (#55890)
openshift/openshift-docs: release_notes/ocp-4-12-release-notes.adoc

==== Updating

To update an existing {product-title} 4.12 cluster to this latest release, see xref:../updating/updating-cluster-cli.adoc#updating-cluster-cli[Updating a cluster using the CLI] for instructions.

[id="ocp-4-12-3"]
=== RHSA-2023:0728 - {product-title} 4.12.3 bug fix and security update

Issued: 2023-02-16

{product-title} release 4.12.3, which includes security updates, is now available. The list of bug fixes that are included in the update is documented in the link:https://access.redhat.com/errata/RHSA-2023:0728[RHSA-2023:0728] advisory. The RPM packages that are included in the update are provided by the link:https://access.redhat.com/errata/RHSA-2023:0727[RHSA-2023:0727] advisory.

You can view the container images in this release by running the following command:

[source,terminal]
----
$ oc adm release info 4.12.3 --pullspecs
----

[id="ocp-4-12-3-bug-fixes"]
==== Bug fixes

> * Previously, when a control plane machine was replaced on a cluster that used the OVN-Kubernetes network provider, the pods related to OVN-Kubernetes sometimes did not start on the replacement machine. When this occurred, the lack of networking on the new machine prevented etcd from allowing it to replace the old machine. As a result, the cluster became stuck in this state and sometimes became degraded. This behavior could occur when the control plane was replaced manually or by the control plane machine set. With this update, the issue is resolved.

Reviewer: Just a suggestion for some bug-doc text wordsmithing to cut down on some of the fluff that was provided with the ccfr. Feel free to tweak it more, but something along the lines of:
```suggestion
* Previously, when a control plane machine was replaced on a cluster that used the OVN-Kubernetes network provider, the pods related to OVN-Kubernetes sometimes did not start on the replacement machine, and prevented etcd from allowing it to replace the old machine.  With this update, pods related to OVN-Kubernetes start in the replacement machine as expected.
```

Author: I've made the suggested changes, thank you!

> (link:https://issues.redhat.com/browse/OCPBUGS-6494[*OCPBUGS-6494*])

Reviewer: Bug link needs to be in line with the end of the bug.
